lib/src/compatKtLint0Dot48Dot0/java/com/diffplug/spotless/glue/ktlint/compat/KtLintCompat0Dot48Dot0Adapter.java

```diff
@@ -97,18 +98,19 @@ public String format(final String text, final String name, final boolean isScrip
 							Collectors.toList()),
 					editorConfigOverrideMap);
 		}
+		EditorConfigDefaults editorConfig;
+		if (editorConfigPath == null || !editorConfigPath.toFile().exists()) {
```

Comment:
Using `java.nio.file.Files.exists(editorConfigPath)` keeps it in the Java NIO domain.